app/assets/javascripts/templates/price_breakdown.html.haml

```diff
@@ -1,4 +1,4 @@
-.joyride-tip-guide.price_breakdown{ng: {class: "{ in: tt_isOpen, fade: tt_animation }", show: "tt_isOpen"}}
+.joyride-tip-guide.price_breakdown{id: "price_breakdown", ng: {class: "{ in: tt_isOpen, fade: tt_animation }", show: "tt_isOpen"}}
```

Comment:
I'm wondering if `.price_breakdown` wasn't enough?

And then, `within(".price_breakdown") do`. What do you think? Why adding a `#` here?